GoogleForCreators/web-stories-wp
# Redesign: border width panel

## Context



## Summary
Redesigns the border panel.


## To-do



## User-facing changes

| Before | After |
|--------|------|
| <img width="297" alt="Screenshot 2021-03-05 at 19 12 19" src="https://user-images.githubusercontent.com/3294597/110149472-5e1b6f00-7dde-11eb-9a77-cfe45a5a1c81.png"> | <img width="297" alt="Screenshot 2021-03-05 at 19 43 28" src="https://user-images.githubusercontent.com/3294597/110152995-b9e7f700-7de2-11eb-9ae9-637018215827.png"> |
| <img width="299" alt="Screenshot 2021-03-05 at 19 12 25" src="https://user-images.githubusercontent.com/3294597/110149488-65427d00-7dde-11eb-9a03-8cbe8214f931.png"> | <img width="294" alt="Screenshot 2021-03-05 at 19 11 02" src="https://user-images.githubusercontent.com/3294597/110152881-958c1a80-7de2-11eb-9d24-44290dcb2331.png"> | 
## Testing Instructions



### QA






This PR can be tested by following these steps:


[Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=4647%3A0)

1. Add a rectangle shape
2. Verify the border panel looks like in Figma
3. Unlock the border width to see all the inputs
4. Add some values to see the color input
5. Verify that all looks as expected.

### UAT







This PR can be tested by following these steps:

1.


## Checklist




---



Fixes #6475
